pytest/tests/loadtest/setup.py

```diff
@@ -0,0 +1,59 @@
+import subprocess
+from time import sleep
+import mocknet_helpers
+import account
+import key
+import argparse
+from os.path import join
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Setup loadtest')
+
+    parser.add_argument('--home', type=str, required=True)
+    parser.add_argument('--num_accounts', type=int, default=5)
+    parser.add_argument('--contract_dir',
+                        type=str,
+                        default='pytest/tests/loadtest/contract')
+    args = parser.parse_args()
+
+    print("Compiling contract")
+    subprocess.check_call(args=[
+        "cargo", "build", "--target", "wasm32-unknown-unknown", "--release"
+    ],
+                          cwd=args.contract_dir)
+
+    validator_key = key.Key.from_json_file(join(args.home,
+                                                "validator_key.json"))
+
+    base_block_hash = mocknet_helpers.get_latest_block_hash()
+    nonce = mocknet_helpers.get_nonce_for_key(validator_key)
+
+    my_account = account.Account(validator_key,
+                                 init_nonce=nonce,
+                                 base_block_hash=base_block_hash,
+                                 rpc_infos=[("localhost", "3030")])
+
+    print(f"Creating {args.num_accounts} accounts.")
+    for i in range(args.num_accounts):
+        account_name = f"shard{i}.test.near"
```

Comment:
How do you ensure that `shard0.test.near` and `shard1.test.near` belong to different shards? Do you define a custom sharding config in `genesis.json`?

Reply:
yes. 

Currently I'm running these tests against the network that I've created using the 'init' call (see the README.md) - and in the recent PR, I've set it up, so that these accounts are showing up on separate shards.

in the future, we should probably do something similar to mocknet